Source/Renderer/ShaderBuilder.js

```diff
@@ -224,6 +226,23 @@ ShaderBuilder.prototype.addAttribute = function (type, identifier) {
   return location;
 };
 
+/**
+ * Add a varying declaration to both the vertex and fragment shaders.
+ *
+ * @param {String} type The GLSL type of the varying
+ * @param {String} identifier An identifier for the varying. Identifiers must begin with <code>v_</code> to be consistent with Cesium's style guide.
+ */
```

Comment:
Add an example in the documentation here, like you've done with `addUniform` and `addAttribute`